libwayshot/src/lib.rs

```diff
@@ -174,18 +167,19 @@ impl WayshotConnection {
         cursor_overlay: i32,
         output: &WlOutput,
         fd: T,
-        capture_region: Option<CaptureRegion>,
-    ) -> Result<FrameFormat> {
+    ) -> Result<(FrameFormat, FrameGuard)> {
```

Comment:
Region is used in recording screen ,I do not think you should remove it

Reply:
You're right, I'll keep support for requesting a screencopy for a specific region for the purposes of screen recording. I hadn't kept that in mind originally.